crowdfunding_compassion/controllers/homepage_controller.py

```diff
@@ -46,28 +47,11 @@ def homepage(self, **kwargs):
 
     @staticmethod
     def _compute_homepage_context():
-        year = datetime.now().year
-        project_obj = request.env["crowdfunding.project"].sudo()
-        current_year_projects = project_obj.get_active_projects(year=year)
-        if len(current_year_projects) < 10 and sum(current_year_projects.mapped("product_number_reached")) < 100:
-            current_year_projects += project_obj.get_active_projects(year=year - 1)
-            year = f"{year-1}/{year}"
-        funds_used = current_year_projects.mapped("product_id")
-        active_funds = funds_used.filtered("activate_for_crowdfunding")
+        # Retrieve all projects open (deadline in the future)
+        current_year_projects = request.env["crowdfunding.project"].sudo().get_active_projects(status='active')
```

Comment:
Maybe a more precise variable name
```suggestion
        active_projects = request.env["crowdfunding.project"].sudo().get_active_projects(status='active')
```